pubsub/rocketmq/metadata.go

```diff
@@ -85,25 +175,44 @@ func (s *rocketMQMetaData) Decode(in interface{}) error {
 	return nil
 }
 
-func parseRocketMQMetaData(metadata pubsub.Metadata, logger logger.Logger) (*rocketMQMetaData, error) {
-	rMetaData := getDefaultRocketMQMetaData()
+func parseRocketMQMetaData(metadata pubsub.Metadata) (*rocketMQMetaData, error) {
+	rMetaData := &rocketMQMetaData{
+		Retries:             3,
+		SendMsgTimeout:      30,
+		LogLevel:            "warn",
+		PullInterval:        100,
+		ConsumerPullTimeout: 30,
+	}
 	if metadata.Properties != nil {
 		err := rMetaData.Decode(metadata.Properties)
 		if err != nil {
 			return nil, fmt.Errorf("rocketmq configuration error: %w", err)
 		}
-	}
-
-	if rMetaData.GroupName != "" {
-		logger.Warn("pubsub.rocketmq: metadata property 'groupName' has been deprecated - use 'producerGroup' instead. See: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-rocketmq/")
-	}
-
-	if rMetaData.SendTimeOut != 0 {
-		logger.Warn("pubsub.rocketmq: metadata property 'sendTimeOut' has been deprecated - use 'sendTimeOutSec' instead. See: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-rocketmq/")
-		if rMetaData.SendTimeOutSec == 0 {
-			rMetaData.SendTimeOutSec = rMetaData.SendTimeOut / 1000000
+		if rMetaData.InstanceName == "" {
+			rMetaData.InstanceName = metadata.Properties["instance"]
```

Comment:
I think this should be `instanceName` ?

Also please define all these property keys as constants.

Reply:
InstanceName has the same effect as instance. Instance is for better compatibility

Reply:
ok, I will fill in the comments